lib/fizzy/parser_expr.cpp

```diff
@@ -94,11 +101,61 @@ parser_result<Code> parse_expr(const uint8_t* pos)
         case Instr::i64_extend_i32_u:
             break;
 
+        case Instr::end:
+        {
+            if (!label_positions.empty())
+            {
+                const auto label_pos = label_positions.back();
+                label_positions.pop_back();
```

Comment:
Hm, why don't we introduce a `stack` type at some point? :)

Reply:
Not a generic one.